proposals/dao/vip_1.ts

```diff
@@ -0,0 +1,59 @@
+import { expect } from 'chai';
+import {
+  DeployUpgradeFunc,
+  NamedAddresses,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+import { getImpersonatedSigner } from '@test/helpers';
+
+/*
+
+Timelock Proposal #1
+
+Description:
+
+Steps:
+  1 - First grant the timelock the governor role by the multisig
+  2 - Grant Timelock PCV Controller
+  3 - Validate state changes
+
+*/
+
+const vipNumber = '1';
+
+// Do any deployments
+// This should exclusively include new contract deployments
+const deploy: DeployUpgradeFunc = async (deployAddress: string, addresses: NamedAddresses, logging: boolean) => {
+  console.log(`No deploy actions for vip${vipNumber}`);
+  return {
+    // put returned contract objects here
+  };
+};
+
+// Do any setup necessary for running the test.
+// This could include setting up Hardhat to impersonate accounts,
+// ensuring contracts have a specific state, etc.
+const setup: SetupUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {
+  const signer = await getImpersonatedSigner(addresses.protocolMultisig);
+  await contracts.core.connect(signer).grantGovernor(addresses.optimisticTimelock);
```

Comment:
Why the need for this `grantGovernor()` call? Should be being done by the proposal no?

Reply:
the multisig has to first grant the timelock the governor role, because the timelock currently does not have that role